### DISCUSSION TITLE
娜艺娜入驻绳网了

### DISCUSSION THREAD

**Discussion by nayina1145141919810** [share121/inter-knot#1887] [其他]
![Uploading IMG_5106.gif…]()

**Comment by nayina1145141919810** [comment]
![Uploading IMG_5106.gif…]()

**Reply by nayina1145141919810** [reply]
我去那咋搞

**Reply by ye018** [reply]
发完图片点一下Preview你那里显示出来图片了在发布

**Reply by nayina1145141919810** [reply]
![IMG_5106](https://github.com/user-attachments/assets/af7edac5-a6ff-45dc-ae96-021cb6435a54)

**Comment by ye018** [comment]
贝如塔

**Comment by ye018** [comment]
![IMG_20240813_171228](https://github.com/user-attachments/assets/5baacbc3-5d31-4c22-9a6b-a6d0b2fe9c71)

**Comment by ye018** [comment]
我是胖猫八月记得给我点麦当劳
![image_download_1723298686931](https://github.com/user-attachments/assets/cc8badb9-dbaf-4d86-b6dc-1777f785ab97)

**Reply by nayina1145141919810** [reply]
其实你是被秦文龙推的

**Reply by ye018** [reply]
其实我并没有死因为肥而不腻

**Comment by nayina1145141919810** [comment]
我服了打不开图片直接给套题🙄💅
[三线合一.pdf](https://github.com/user-attachments/files/16597792/default.pdf)

**Reply by ye018** [reply]
什金😹👌

**Reply by nayina1145141919810** [reply]
月半猫我错了😭

**Reply by ye018** [reply]
知道错了就好

**Comment by Dave-12138** [comment]
>我服了打不开图片

你看见有uploading就说明图还没上传完，要等上传完再点提交

**Reply by nayina1145141919810** [reply]
好的

**Reply by nayina1145141919810** [reply]
可以了多谢大佬关心

**Comment by Dave-12138** [comment]
![图片](https://s2.loli.net/2024/08/13/ynSD7l8E6tcbYUI.webp)
右上角三个点，点edit可以修改你发的内容

**Reply by nayina1145141919810** [reply]
我服了这手机端有点慢

**Reply by ye018** [reply]
@nayina1145141919810 使用魔法

**Comment by shenguojia123** [comment]
疑似新型以骸进入绳网，各位绳斤要小心口牙！
